examples/httpserver_cpython.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2024 Ted Timmons
+#
+# SPDX-License-Identifier: Unlicense
+#
+# Simple example showing how to use httpserver in cpython (e.g., on a computer).
+
+import socket
+from adafruit_httpserver import Server, Request, Response
+
+
+# fake the wifi class, reduces changes required later in the code.
+class wifi:  # pylint: disable=too-few-public-methods
```

Comment:
I think it is not necessary to create such class, I believe simply passing `"0.0.0.0"` as argument to `server.serve_forever()` is enough. The only reason other examples use `wifi.radio.ipv4_address` is because this is the way CircuitPython handles that.